makelib/uptest.mk

```diff
@@ -0,0 +1,66 @@
+# ====================================================================================
+# End to End Testing
+
+UPTEST_EXTRA_ARGS ?=
+
+UPTEST_COMMAND = SKIP_DEPLOY_ARGO=$(SKIP_DEPLOY_ARGO) \
+	KUBECTL=$(KUBECTL) \
+	CHAINSAW=$(CHAINSAW) \
+	CROSSPLANE_CLI=$(CROSSPLANE_CLI) \
+	CROSSPLANE_NAMESPACE=$(CROSSPLANE_NAMESPACE) \
+	YQ=$(YQ) \
+	$(UPTEST) e2e $(UPTEST_CLAIMS) \
+	--data-source="${UPTEST_DATASOURCE_PATH}" \
+	--setup-script=$(UPTEST_SETUP_SCRIPT) \
+	--default-timeout=2400s \
+	--skip-update \
+	--skip-import \
```

Comment:
These are also mainly special for the configuration/claim tests. It is again not default for the provider repositories. We may consider having a different name for the `UPTEST_COMMAND` to `UPTEST_CONFIGURATION_COMMAND`.

And with related to the above command, we can prevent the `UPTEST_CLAIMS` variable with this way and change the name of the `UPTEST_COMMAND` and `uptest` target to the configuration specific names.